install/linux/usr/share/odemis/sim/meteor-fibsem-sim.odm.yaml

```diff
@@ -80,8 +80,9 @@ METEOR-FIBSEM-Sim: {
         SAMPLE_CENTERS: {"GRID 1": {'x': 0, 'y': 0, 'z': 32.e-3}, "GRID 2": {'x': 5.0e-3, 'y': 0, 'z': 32.e-3}},
         # Mirroring values between SEM - METEOR
         POS_COR: [0.0253126, 0.0024916],
-        CALIB: {"version": "tfs_2", 
-                "Sample pre-tilt":  0.6108652381980153},  # 35°
+        CALIB: {"version": "tfs_3", 
+                "Sample pre-tilt":  0.6108652381980153, # 35°
+                "SEM-Eucentric-Focus": 7.0e-3},  # aqulios=7.0e-3, hydra=4.0e-3
```

Comment:
That file doesn't start in the GUI:
```2025-02-07 15:17:41,052 ERROR   main:438:       [CAP] self.check_calib_data(required_keys={model.MD_SAMPLE_PRE_TILT, "dx", "dy"}) # TODO: add "SEM-Eucentric-F
ocus"
2025-02-07 15:17:41,052 ERROR   main:438:       [CAP] File "/home/piel/development/odemis/src/odemis/acq/move.py", line 224, in check_calib_data
2025-02-07 15:17:41,052 ERROR   main:438:       [CAP] raise ValueError(f"Stage metadata {model.MD_CALIB} is missing the following required keys: {missing_keys}.")
```

Reply:
I've updated it, but that isn't a valid config for this yet (no fibsem gui), please look at https://github.com/delmic/odemis/pull/3005 first